tests/acceptance/features/bootstrap/ChecksumContext.php

```diff
@@ -223,48 +225,56 @@ public function theWebdavChecksumShouldMatch(string $expectedChecksum):void {
 			__METHOD__ . " parsed XML parsed0 does not have key value"
 		);
 		$parsed0Value = $parsed0['value'];
+		Assert::assertIsArray($parsed0Value, __METHOD__ . " parsed XML parsed0Value is not an array");
```

Comment:
IMO those error messages will not help when seeing them in a test-run output.
```suggestion
		Assert::assertIsArray($parsed0Value, __METHOD__ . " could not find checksum in parsed XML expecting `$parsed[0]['value']` to be an array but received '" . $parsed0Value . "'");
```

similar in the following code

Reply:
fixed